lib/db/schema.sql

```diff
@@ -18,6 +18,7 @@ CREATE TABLE IF NOT EXISTS codes (
   INDEX codes_user_id(userId),
   email VARCHAR(256) NOT NULL,
   scope VARCHAR(256) NOT NULL,
+  lastAuthAt BIGINT NOT NULL,
```

Comment:
I'm hesitant to do any transformation of coercion on this value on the off change that we stuff it up; `BIGINT` is what the auth-server uses for this value so I just copied it across here.